Composer/packages/tools/language-servers/language-understanding/src/LUServer.ts

```diff
@@ -101,6 +107,36 @@ export class LUServer {
     this.connection.listen();
   }
 
+  protected definitionHandler(params: TextDocumentPositionParams): Location | undefined {
```

Comment:
this is not in parity with what Lg does, in Lg you also had the lineNumber, is there any reason you dont have it here?

Reply:
In LU editor, the goto definition only needs to handle the navigation of import another LU file or another intent from other files. For example, we only need to extract the target file and section as the lines below.
```
[import add item](./additem.en-us.lu#TextInput_Response_282825)
[import delete item](./deleteitem.en-us.lu)
```
So there is no goto between different lines in the same file.
Also the signature of onNavigateToLuPage is `(luFileId: string, luSectionId?: string | undefined) => void)`, that's why I did not use lineNumber here.

Reply:
I was under the impression that you'll have go to definition for entity as well.